test/unit/calculators/register_a_birth_rates_query_test.rb

```diff
@@ -6,31 +6,20 @@ class RegisterABirthRatesQueryTest < ActiveSupport::TestCase
       @rates_query = RatesQuery.from_file("register_a_birth")
     end
 
-    context "for 2015/16" do
+    context "The current and last dates in the file" do
       setup do
-        @rates = @rates_query.rates(Date.parse("2015-04-06"))
+        @rate_array = [
+          @rates_query.rates,
+          @rates_query.rates(Date.parse("2099-01-01")),
```

Comment:
should this be `2999`?

Reply:
It should be! Thanks. (and the equivalent _deaths one).